spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe RegisteredApplicationsController, type: :controller do
+
+  describe "GET#index" do
+    let(:my_user) { create(:user) }
```

Comment:
Try to avoid the `my_` prefix for names of variables, methods, etc. unless it adds clarity.